Source/santasyncservice/SNTSyncPreflight.mm

```diff
@@ -135,7 +135,11 @@ - (BOOL)sync {
       resp.deprecated_transitive_whitelisting_enabled());
   self.syncState.enableAllEventUpload = @(resp.enable_all_event_upload());
   self.syncState.disableUnknownEventUpload = @(resp.disable_unknown_event_upload());
-  self.syncState.eventBatchSize = resp.batch_size();
+
+  self.syncState.eventBatchSize = 50;
```

Comment:
Is this some known/previous default? Can we make it a constant and comment it?

Reply:
Done